pkg/sdk/testint/grants_integration_test.go

```diff
@@ -205,6 +206,83 @@ func TestInt_GrantAndRevokePrivilegesToAccountRole(t *testing.T) {
 		require.NoError(t, err)
 		assert.Equal(t, 0, len(grants))
 	})
+
+	t.Run("grant and revoke on all pipes", func(t *testing.T) {
```

Comment:
can we have a negative test that will show that multiple errors are returned?

Reply:
Done. This change made me think about how we produce random names for our resources and that sometimes this is the cause of non-stable tests. In this case, I saw that pipes require identifiers within a certain range of characters and all of our random ID generators were not satisfying its requirements. This pushed me to make my own simple random name generator with an a-z set of characters (other ones like String, StringN, AlphaN, AlphanumericN return unsupported characters).